### PR TITLE
fix the typelist service issue

### PR DESCRIPTION
**Pull request by maielgendy**

### CODE REVIEW

File: nfafrontend/src/app/shared/data-storage.service.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import {Nfa} from './nfa.model';
 import {Injectable} from '@angular/core';
 import {Http} from '@angular/http';
 import {Project} from './project.model';
+import {Type} from './type.model';
```
Comment (HarishVijayamohan) [inline nfafrontend/src/app/shared/data-storage.service.ts]: this file is missing in the folder

File: nfafrontend/src/app/shared/data-storage.service.ts [hunk 2]
```diff
@@ -13,12 +14,11 @@ export class DataStorageService {
     console.log(nfa);
    return this.http.post('http://localhost:8080/nfabackend/webapi/nfa_catalog/' , nfa);
   }
-  storeProject(newproject: Project) {
+  storeProject(newproject: Project, type: Type[]) {
     console.log(newproject);
-    return this.http.post('http://localhost:8080/nfabackend/webapi/project/create', newproject);
+    return this.http.post('http://localhost:8080/nfabackend/webapi/project/create', newproject:type);
```
Comment (HarishVijayamohan) [inline nfafrontend/src/app/shared/data-storage.service.ts]: I get some error here saying - 
ERROR in src/app/shared/data-storage.service.ts(19,95): error TS1005: ',' expected.

Comment (jmnietsch) [reply nfafrontend/src/app/shared/data-storage.service.ts]: I have the same error. @maielgendy Please try to check this out as a new repository and see if it builds then.

**Comment by jmnietsch** [comment]
Remerged and will be proposed again

**Pull request closed without merging**